app/src/main/res/layout/view_switchcompat_preference.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Can probably safely name these preferences with _switch_. It's not like we'll ever use the non-compat Switch.

Reply:
I actually didn't do this initially because then the `preferenceCheckableType` attribute becomes inconsistent. Currently it allows for `check_box` or `switch_compat`. I can't change the second one to `switch` because that's a Java reserved word... But if you want I can still edit the view layout itself to just be `view_switch_preference`

Reply:
✔️

Reply:
Yeah, I think it's ok to have the layout files and such named `switch` alone and that type attr can be left as `switch_compat`.